.github/workflows/connector-ci-tests.yml

```diff
@@ -68,6 +68,10 @@ jobs:
   jvm-connectors-test-matrix:
     needs: [generate-matrix]
     runs-on: linux-24.04-large # Custom runner, defined in GitHub org settings
+    concurrency:
+      # Large runners aren't free, so cancel in-progress jobs when a new commit is pushed to the PR.
+      cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
```

Comment:
Actually, I think this might limit this to one connector run at a time.

Reply:
it should be `group: ${{ github.workflow }}-jvm-tests-${{ matrix.connector }}-${{ github.head_ref }}`

Reply:
Fixed - thanks!

Reply:
Actually - now I'm not sure if this should be evaluated _prior_ to the matrix context evaluation or _before_ it. I'll kick off a connector test to confirm.